app/controllers/application_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery prepend: true
```

Comment:
This one was interesting. We call `skip_before_action :verify_authenticity_token` in `UnauthenticatedApiController`, but nothing ever actually defined a `verify_authenticity_token` before_action. In Rails 4, skipping an undefined filter method would silently do nothing, but on Rails 5 it now raises an exception.

So I could either remove the `skip_before_action :verify_authenticity_token` line (all tests pass if I do that), or add this line which actually does add a `verify_authenticity_token` method for us to skip. 😅

Reply:
seems good.